fees/gyroscope/index.ts

```diff
@@ -0,0 +1,40 @@
+import { Adapter, FetchOptions } from "../adapters/types";
+import { CHAIN } from "../helpers/chains";
+import { Chain } from "@defillama/sdk/build/general";
+import { queryDune } from "../helpers/dune";
+
+const fetch = (_: Chain) => {
+  return async (options: FetchOptions) => {
+    const dailyFees = options.createBalances();
+    try {
+      const value = await queryDune("4776022", {});
```

Comment:
seem your query is wrong
1. should split by chain

Reply:
also, can you compute this on chain using event logs, the dune query is bit confusing